src/Compilers/Core/RebuildTest/OptionRoundTripTests.cs

```diff
@@ -10,12 +10,12 @@
 using System.Reflection.PortableExecutable;
 using System.Text;
 using System.Threading;
-using BuildValidator;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
 using Microsoft.CodeAnalysis.Emit;
 using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Rebuild;
 using Microsoft.CodeAnalysis.VisualBasic;
```

Comment:
nit: swap to keep ordered alphabetically.

```suggestion
using Microsoft.CodeAnalysis.Rebuild;
using Microsoft.CodeAnalysis.Test.Utilities;
using Microsoft.CodeAnalysis.VisualBasic;
```